packaging/debian/rules

```diff
@@ -20,12 +20,15 @@ PKGDIR = debian/storagenode
 
 override_dh_auto_install:
 # downloaded binaries directory
-	mkdir -p ${PKGDIR}/var/storagenode
+	mkdir -p ${PKGDIR}/var/lib/storagenode
```

Comment:
If we aren't putting anything in these directories while building the package, these lines won't have any effect. Debs, like git, ignore empty directories.

Reply:
I think I was wrong about this. Maybe it's been too long.